config/Dockerfiles/cloud-image-compose/virt-customize.sh

```diff
@@ -109,7 +109,8 @@ virt-copy-in -a ${DOWNLOADED_IMAGE_LOCATION} ${virt_copy_files} /etc/yum.repos.d
 if [ "${namespace}" != "tests" ]; then
     for pkg in $(repoquery -q --disablerepo=\* --enablerepo=${package} --repofrompath=${package},${rpm_repo} --all --qf="%{ARCH}:%{NAME}" | sed -e "/^src:/d;/-debug\(info\|source\)\$/d;s/.\+://" | sort -u) ; do
         # check if this package conflicts with any other package from RPM_LIST
-        conflict=$(repoquery -q --disablerepo=\* --enablerepo=${package} --repofrompath=${package},${rpm_repo} --conflict $pkg | awk '{print$1}')
+        conflict_capability=$(repoquery -q --disablerepo=\* --enablerepo=${package} --repofrompath=${package},${rpm_repo} --conflict $pkg)
+        conflict=$(repoquery -q --disablerepo=\* --enablerepo=${package} --repofrompath=${package},${rpm_repo} --whatprovides "$conflict_capability" | awk '{print$1}')
```

Comment:
The result of this query is the NEVRA of package that matches so `awk '{print$1}` also just prints out the NEVRA.  I think the the comparison on line 117 is just comparing the package name so we probably want to add -`-qf "%{NAME}"` to this query.

Reply:
For some reason this did not show up for me until now... Looking. I will open another PR if needed

Reply:
Yes, I agree. Opening another PR